src/cider/nrepl/middleware/debug.clj

```diff
@@ -182,8 +182,10 @@ this map (identified by a key), and will `dissoc` it afterwards."}
 (defn pr-short
   "Like `pr-str` but limited in length and depth."
   [x]
-  (binding [*print-length* (:length @print-options)
-            *print-level*  (:level @print-options)]
+  (binding [*print-length* (or (:length @print-options)
```

Comment:
I'm wondering why `print-options` is some var, as if I recall correctly it's passed as part of each nREPL message. Most likely it's not initialized properly. Perhaps we broke something when we introduced support for nREPL 0.6.